packages/key-management/src/util/ownSignatureKeyPaths.ts

```diff
@@ -68,18 +94,25 @@ export const ownSignatureKeyPaths = async (
   txBody: Cardano.TxBody,
   knownAddresses: GroupedAddress[],
   inputResolver: Cardano.util.InputResolver
-): Promise<AccountKeyDerivationPath[]> => {
+): Promise<SignatureKeyPathsResult> => {
+  let partialSign = false;
+
   const paymentKeyPaths = uniq(
     (
       await Promise.all(
         txBody.inputs.map(async (input) => {
           const ownAddress = await inputResolver.resolveInputAddress(input);
           if (!ownAddress) return null;
```

Comment:
See comment above about renaming argument to `ownInputResolver`. Let's add a doc comment about the input resolver argument to document it's expected behavior.
```suggestion
          if (!ownAddress) {
            partialSign = true;
            return null;
          }
```